crates/env/src/engine/on_chain/ext.rs

```diff
@@ -626,3 +630,13 @@ pub fn ecdsa_recover(
     };
     ret_code.into()
 }
+
+pub fn is_contract(account_id: &[u8]) -> bool {
+    let ret_val = unsafe { sys::seal_is_contract(Ptr32::from_slice(account_id)) };
+    !(ret_val.0 == 0)
```

Comment:
I'd change this to something like:

```rust
let res: Result = ret_val.into();
res.is_ok()
```

It's easier to read

Reply:
refactored to look like this